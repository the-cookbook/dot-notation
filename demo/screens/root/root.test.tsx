import * as React from 'react';
import renderer from 'react-test-renderer';

import Root from './root';

describe('screens/<Root />', () => {
  it('should match snapshot', () => {
    const wrapper = renderer.create(<Root />);

    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
