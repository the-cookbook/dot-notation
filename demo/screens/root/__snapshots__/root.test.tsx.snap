// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`screens/<Root /> should match snapshot 1`] = `
<div
  className="container-fluid container full-height"
>
  <div
    className="row full-height"
  >
    <div
      className="content"
    >
      <div
        className="content-wrapper full-height"
      />
    </div>
  </div>
</div>
`;
